NEWS.md

```diff
@@ -32,6 +32,7 @@
 
 ## New and improved features
 
+* `seq_linter()` now includes lints to inform about missed opportunities to use the `sequence()` base R function (#2618, @Bisaloo)
```

Comment:
I think `sequence()` is not a very well-known function ([<500 calls on CRAN vs more than 80K for `seq()`](https://github.com/search?q=org%3Acran+lang%3AR+%2F%5B%5E0-9a-z._%5Dsequence%5B%28%5D%2F&type=code)), here is a good opportunity to quickly introduce it:

"to use the `sequence()` function, e.g. `unlist(lapply(ints, seq))`"